java/arcs/core/data/testdata/BUILD

```diff
@@ -21,8 +22,14 @@ arcs_manifest_proto(
     visibility = ["//visibility:public"],
 )
 
+arcs_kt_schema(
+    name = "example_schema",
+    srcs = ["WriterReaderExample.arcs"],
+)
+
 arcs_kt_plan(
     name = "example_plan",
     src = "WriterReaderExample.arcs",
```

Comment:
Would be nice if this could accept a list of srcs like arcs_kt_schema

Reply:
Thanks! I forgot about this one; implemented.